integrations/jira/.port/spec.yaml

```diff
@@ -28,6 +29,11 @@ configurations:
     type: string
     description: You can configure the user token on the <a target="_blank" href="https://id.atlassian.com/manage-profile/security/api-tokens">Atlassian account page</a>
     sensitive: true
+  - name: atlassianOrganizationId
+    required: false
+    type: string
+    description: Read <a target="_blank" href="https://confluence.atlassian.com/jirakb/what-it-is-the-organization-id-and-where-to-find-it-1207189876.html">this</a> to find your Atlassian Organization ID
+    sensitive: false
```

Comment:
I think we need to describe what its used for in the integration as its not clear ATM